pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/Namespaces.java

```diff
@@ -1489,7 +1489,7 @@ public void deleteCompactionThreshold(@PathParam("tenant") String tenant,
                   notes = "A negative value disables automatic offloading")
     @ApiResponses(value = { @ApiResponse(code = 403, message = "Don't have admin permission"),
                             @ApiResponse(code = 404, message = "Namespace doesn't exist") })
-    public long getOffloadThreshold(@PathParam("tenant") String tenant,
+    public Long getOffloadThreshold(@PathParam("tenant") String tenant,
```

Comment:
Maybe we need to think about why the ``internalGetOffloadThreshold method returns null``  , because null is an incorrect result.

Reply:
same

Reply:
But this will break the PulsarAdmin?

Reply:
pulsar admin client will get internal server error.

Reply:
Sorry I mean, after introduces this change, the Admin will get an NPE? The Admin also define the return type as `long`

```
long getOffloadThreshold(String namespace) throws PulsarAdminException;
```

Reply:
We also need make a change here.

Reply:
@gaozhangmin 

I'm very confused, what does it mean to return null if an api needs to return a number.

Reply:
Can we try to return a non-null value to the client? Otherwise, we might introduce regression and compatibility issues after users upgrade to the server to a new version but the Admin is an old version.

Reply:
+1, people can use `pulsar-admin` not only in the terminal, but  the code